templates/READMEs/README.md

```diff
@@ -1,107 +1,133 @@
-<!-- TODO: This is a template SDK README.md. It might also be useful for other repository types. Update the TODOs with correct examples/documentation. Replace all instances of "your-repo" with the appropriate name -->
+<!-- TODO: This is a template SDK README.md.
+It's structured to fit nicely into openfeature.dev, which is configured to include SDK READMEs.
+Complete all the TODOs applicable for your implementation and then create an issue in https://github.com/open-feature/openfeature.dev
```

Comment:
Please add a snippet that links to the section of the docs where you can add the SDK.

https://github.com/open-feature/openfeature.dev/blob/main/scripts/sdks.ts

Reply:
I was hesitant to do this because I though it might be brittle (if that file is moved, for instance) so I just described that the content is included and encouraged creating an issue.